core/src/avm2/globals/flash/utils.rs

```diff
@@ -140,6 +140,56 @@ pub fn escape_multi_byte<'gc>(
     Ok(v.into())
 }
 
+fn handle_percent<I>(it: &mut I) -> Option<u8>
+where
+    I: Iterator<Item = char>,
+{
+    let mut d = it.next()?.to_digit(16)?;
+    d = d * 16 + it.next()?.to_digit(16)?;
+    Some(d as u8)
+}
+
+/// Implements `flash.utils.unescapeMultiByte`
+pub fn unescape_multi_byte<'gc>(
+    activation: &mut Activation<'_, 'gc, '_>,
+    _this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error> {
+    let s = args
+        .get(0)
+        .unwrap_or(&Value::Undefined)
+        .coerce_to_string(activation)?;
+    let bs = s.as_wstr();
+    let mut buf = WString::new();
+    let mut it = bs.chars().map(|c| c.unwrap_or(char::REPLACEMENT_CHARACTER));
```

Comment:
nit: Let's rename `it` to `chars`, to be consistent with `effective_replacement` in `regexp.rs`.

Reply:
Done.